### PR TITLE
feat: add check for availability of cluster port before `project up`

### LINKED ISSUE #241: check if port(s) already allocated on project up
If port(s) are already allocated, which are needed in `project up` stop process and display an error.

### PR DESCRIPTION
**Pull request by SteinRobert**
Fixes #241 

Adds a simple check for the availability of the cluster port retrieved via GraphQL.

### CODE REVIEW

File: src/cli/helper.py [hunk 1]
```diff
@@ -1,5 +1,28 @@
...
+    return result_of_check == 0
+
+
+def check_ports(port_list: List[int]):
+    """
+    Takes a list of integers and check whether those ports are available.
+
+    Returns a list of ports which are busy.
+    """
+    return list(filter(lambda port: port_in_use(port), port_list))
```
Comment (schwobaseggl) [inline src/cli/helper.py:25]:
```python
return list(filter(port_in_use, port_list))
```

**Review by schwobaseggl — CHANGES_REQUESTED** [review]
One minor simplification

File: src/cli/helper.py (hunk 1 above)
Comment (SteinRobert) [reply src/cli/helper.py:25]: Hahaha, fair enough. I'll tackle it.

Comment (SteinRobert) [reply src/cli/helper.py]: Done.

**Review by schwobaseggl — APPROVED** [review]

**Pull request merged**